class.jetpack-options.php

```diff
@@ -261,7 +327,13 @@ public static function delete_option( $names ) {
 	}
 
 	private static function get_grouped_option( $group, $name, $default ) {
-		$options = get_option( self::$grouped_options[ $group ] );
+		if ( self::is_network_option( $name ) ) {
```

Comment:
So, my point here is that the `file_data` option we're changing is a non-grouped option, so any changes to enable network storage of grouped option won't have any bearing on this use case.

I can understand potentially adding it for API Consistency or the like, but handling grouped options may be tricky, as  it may be desirable to flag some of the options as network and some not -- so for the moment I'd prefer not to add any functionality to let grouped options be flagged as network storage.

Does that make sense?